gyp_xwalk

```diff
@@ -351,6 +351,10 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if gyp_vars_dict.get('OS') == 'win':
+    args.append('-Denable_widevine=1')
+    args.append('-Denable_pepper_cdms=1')
```

Comment:
@halton Do you mean like this to add -Denable_widevine=1 and -Denable_pepper_cdms=1?

I tried it. But it seems not work. My steps are using current code. Then run:
1. python gyp_xwalk.py -Dmediacodecs_EULA=1
2. ninja build
but the video can't be played.
If I use 'python gyp_xwalk.py -Denable_widevine=1 -Denable_pepper_cdms=1 -Dmediacodecs_EULA=1', then build. The video can be played well. Seems that this setting doesn't work. Is anything wrong?